src/routes/test/[slug]/api/submit-answer/+server.ts

```
@@ -84,11 +69,12 @@ export const POST: RequestHandler = async ({ request, cookies }) => {
 			try {
 				const errorData = await res.json();
 				errorMessage = errorData?.detail || errorData?.message || errorData?.error || errorMessage;
-			} catch {
-				// Keep the status-based fallback when the backend error body is not JSON.
-			}
+			} catch {}
```

ESLint

[error] Empty block statement.

(no-empty)